src/Modules/main_class.js

```diff
@@ -0,0 +1,113 @@
+import trash from '../trash.png';
+import addTask from './add_task.js';
+import removeTask from './remove_task.js';
+import editTask from './edit_task.js';
+import markComplete, { deleteComplete } from './complete_task.js';
+
+const add = addTask;
+const removeone = removeTask;
+const editone = editTask;
+const complete = markComplete;
+const dropComplete = deleteComplete;
+
+export default class Taks {
+  static init() {
+    const newList = new Taks();
+    return newList;
+  }
+
+  constructor() {
+    document.querySelector('form').addEventListener('submit', (e) => {
+      e.preventDefault();
+      this.addNewTasks();
+    });
+
+    this.tasks = [];
+    this.name = 'ListTasks';
+    this.loadTasks();
+    this.tableTask();
+  }
+
+  addNewTasks = () => {
+    add(this.tasks, this.saveTask, this.loadTasks);
+    this.tableTask();
+  }
+
+  tableTask = () => {
+    const div = document.querySelector('.contain--tasks');
+    const buttondrop = document.querySelector('.drop');
+
+    const ul = document.createElement('ul');
+    div.innerHTML = '';
+    this.tasks.forEach((elem, index) => {
+      const taskli = document.createElement('li');
+      taskli.className = 'flexrow pading';
+      const inputTask = document.createElement('input');
+      inputTask.type = 'checkbox';
+      const labelTask = document.createElement('input');
+      labelTask.className = 'flexrow text taskdesc';
+      labelTask.value = elem.description;
+      const imgtrash = document.createElement('img');
+      imgtrash.src = trash;
+      imgtrash.alt = 'delete';
+      imgtrash.className = 'trash';
+      taskli.appendChild(inputTask);
+      taskli.appendChild(labelTask);
+      taskli.appendChild(imgtrash);
+
+      if (elem.completed === false) {
+        inputTask.checked = false;
+      } else {
+        inputTask.checked = true;
+      }
```

Comment:
Hey, your logic is very well thought out however this code could be rewritten in a cleaner way, in one line of code, something like this:  `inputTask.checked = elem.completed`

Reply:
Hi @NitBravoA92, If I understand you suggested me to erase that part of the code and replace it for `inputTask.checked = elem.completed`, isn't it?

Reply:
Hi @AndreaM2429 Yes, it's just a suggestion to make the code more compact and clean.

Reply:
Thanks for that, I appreciate this a lot